posthog/api/organization_feature_flag.py

```diff
@@ -27,6 +31,75 @@ def retrieve(self, request, *args, **kwargs):
         teams = self.organization.teams.all()
 
         flags = FeatureFlag.objects.filter(key=feature_flag_key, team_id__in=[team.id for team in teams])
-        flags_data = [{"team_id": flag.team_id, "active": flag.active} for flag in flags]
+        flags_data = [
+            {
+                "flag_id": flag.id,
+                "team_id": flag.team_id,
+                "active": flag.active,
+            }
+            for flag in flags
+        ]
 
         return Response(flags_data)
+
+    @action(detail=False, methods=["post"], url_path="copy_flags")
+    def copy_flags(self, request, *args, **kwargs):
+        body = request.data
+        feature_flag_key = body.get("feature_flag_key")
+        from_project = body.get("from_project")
+        target_project_ids = body.get("target_project_ids")
+
+        if not feature_flag_key or not from_project or not target_project_ids:
+            return Response({"error": "Missing required fields"}, status=status.HTTP_400_BAD_REQUEST)
+
+        try:
+            flag_to_copy = FeatureFlag.objects.get(key=feature_flag_key, team_id=from_project)
+        except FeatureFlag.DoesNotExist:
+            return Response({"error": "Feature flag to copy does not exist."}, status=status.HTTP_404_NOT_FOUND)
+
+        results = {"success": [], "failed": []}
+
+        for target_project_id in target_project_ids:
+            # Target project does not exist
+            try:
+                Team.objects.get(id=target_project_id)
+            except ObjectDoesNotExist:
+                results["failed"].append({"project_id": target_project_id, "errors": "Target project does not exist."})
+                continue
+
+            context = {
+                "request": request,
+                "team_id": target_project_id,
+            }
+            flag_data = {
+                "key": flag_to_copy.key,
+                "name": flag_to_copy.name,
+                "filters": flag_to_copy.filters,
```

Comment:
dicey, we're missing important properties

"rollout_percentage" (for old flags compatibility)
"ensure_experience_continuity"

also add a comment in the model that when new fields are added, check whether they need to be added here.